deployment/kubernetes/helm/pulsar/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
Would it be better to match the project version here as well?

Reply:
Again, not sure what's the best practice in Helm, whether the same Helm chart can work for different version of the images or if we need to update the chart version in order to upgrade Pulsar version.

Reply:
I think in this case it would be a "personal" preference for the pulsar project. As described here https://github.com/kubernetes/helm/blob/master/docs/charts.md#the-appversion-field this is an informational field with no impact in deployments. Do you want it changed?